src/home_robot/home_robot/mapping/semantic/instance_tracking_modules.py

```diff
@@ -154,10 +156,19 @@ class InstanceMemory:
     unprocessed_views: List[Dict[int, InstanceView]] = []
     timesteps: List[int] = []
 
-    def __init__(self, num_envs: int, du_scale: int, debug_visualize: bool = False):
+    def __init__(
```

Comment:
can we change the instance_map, segmentation_map names to something more generic + add a docstring? also make segmentation optional

Reply:
Changed to instance_seg, semantic_seg. Sorry for the confusion.